modules/bot/controllers/privates/GroupGuestController.php

```diff
@@ -491,4 +501,33 @@ public function actionMembersWithReviews($id = null, $page = 1)
             )
             ->build();
     }
+
+    /**
+     * @param int $id ChatMember->id
+     *
+     * @return array
+     */
+    public function actionRemoveMembership($id = null)
+    {
+        $chatMember = ChatMember::findOne($id);
+
+        if (isset($chatMember) && $chatMember->getUserId() == $this->getTelegramUser()->getId()) {
```

Comment:
if (!isset($chatMember) || ($chatMember->getUserId() != $this->getTelegramUser()->getId())) {
        return $this->getResponseBuilder()
             ->answerCallbackQuery()
             ->build();
}